detection/evasion/unexpected-alf-exceptions-macos.sql

```diff
@@ -46,20 +46,20 @@ WHERE -- Filter out stock exceptions to decrease overhead
   ) -- Ignore files that ahve already been removed
   AND file.filename NOT NULL
   AND exception_key NOT IN (
+    ',,/Users/cpanato/code/src/github.com/sigstore/docs/node_modules/.bin/hugo/hugo,501',
```

Comment:
We should probably de-personalize cases like this in a future PR. Admittedly, I didn't know we had these.

Reply:
Done in #449